downstream/modules/platform/proc-controller-configure-secret-lookups.adoc

```diff
@@ -0,0 +1,69 @@
+[id="proc-controller-configure-secret-lookups"]
+
+= Configure and link secret lookups
+
+When pulling a secret from a 3rd-party system, you are in essence linking credential fields to external systems. 
+To link a credential field to a value stored in an external system, select the external credential corresponding to that system and provide `metadata` to look up the desired value. The metadata input fields are part of the `external credential type` definition of the `source credential`.
+
+{ControllerNameStart} provides a `credential plugin` interface for developers, integrators, system administrators, and power-users with the ability to add new external credential types to extend it to support other secret management systems. 
+
+For more information, see the development documents for
+link:https://github.com/ansible/awx/blob/devel/docs/credentials/credential_plugins.md[Credential plugins].
+
+Use the following procedure to use {ControllerName} to configure and use each of the supported 3-party secret management systems.
+
+.Procedure
+. Create an external credential for authenticating with the secret management system. At minimum, provide a name for the external
+credential and select one of the following for the *Credential Type*:
+
+* xref:ref-controller-metadata-credential-input[Metadata for credential input sources]
+* xref:ref-centrify-vault-lookup[Centrify Vault Credential Provider Lookup]
+* xref:ref-cyberark-ccp-lookup[CyberArk Central Credential Provider (CCP) Lookup]
+* xref:ref-cyberark-conjur-lookup[CyberArk Conjur Secrets Manager Lookup]
+* xref:ref-hashicorp-vault-lookup[HashiCorp Vault Secret Lookup]
+* xref:ref-hashicorp-signed-ssh[HashiCorp Vault Signed SSH]
+* xref:ref-azure-key-vault-lookup[{Azure} Key Vault]
+* xxref:ref-thycotic-devops-vault[Thycotic DevOps Secrets Vault]
+* xref:ref-thycotic-secret-server[Thycotic Secret Server]
+
+. Navigate to the credential form of the target credential and link one or more input fields to the external credential along with metadata for locating the secret in the external system. 
+In this example, the _Demo Credential_ is the target credential.
+. For any of the fields below the *Type Details* area that you want to link to the external credential, click the image:leftkey.png[Link,15,15] icon in the input field. 
```

Comment:
For any of the fields following... or For any of the fields that follow Type Details...